lib/stages/pulsarPostProcess.cpp

```diff
@@ -188,17 +195,11 @@ void pulsarPostProcess::main_thread() {
         if (!new_frame_fpga_seq_num.has_value())
             return;
 
+        // Initialize data for header info, namely position and frequency labels.
+        // See fill_headers() for bit-packing of freq labels into PSRHeader.eud3.
```

Comment:
Could a doxygen comment for `PSRHeader` struct explain how the labels are packed into eud3?

Reply:
never used `doxygen` before but i took a stab at a doxygen-readable comment.